tfutils/base.py

```diff
@@ -334,10 +336,6 @@ def run(sess,
         - thres_loss (float, default: 100)
             If loss exceeds this during training, HiLossError is thrown
     """
-    # initialize and/or restore variables for graph
```

Comment:
Yeah, I was confused why initialization disappeared from here and first thought about asking you but then decided it must have been a mistake. The reason I put it back was because an error was raised at line 349 `step = global_step.eval(session=sess)`: `global_step` could not be evaluated since it has not been initialized. Not sure now why this was happening; will have to look again on Monday.